CorsixTH/Lua/world.lua

```diff
@@ -551,9 +551,10 @@ function World:spawnVIP(name)
   vip.enter_deaths = hospital.num_deaths
   vip.enter_visitors = hospital.num_visitors
   vip.enter_cures = hospital.num_cured
-
-  vip.enter_explosions = hospital.num_explosions
-
+  vip.enter_patients = #hospital.patients
+  if #self.rooms > 79 then
```

Comment:
A couple magic numbers here, please document them (and/or use local 'constants' with names so they are self documenting)